kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -1,11 +1,20 @@
-{% extends "kolibri/base.html" %}
```

Comment:
@benjaoming has done some similar refactor in his branch, bundling together both the sync and async assets into a single tag, so it may be good to hold off on too much refactor until that is integrated.